docs/guide/ai-extension-model-config.md

```
@@ -1,82 +1,107 @@
+
+![添加大模型配置](../assets/images/guide/addModel.gif)
+
+> 如果你需要的模型不在列表中，请让开发者参考下方的"工程配置"添加。
+
+## 二、工程配置（开发者）
+
+### 目录结构
+
+在扩展工程 `next-wxt` 中，`entrypoints/sidepanel/model-manage` 文件夹负责管理模型配置：
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)